.github/workflows/template.bicep.test.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         with:
           creds: "${{ secrets.azureEnviromentCredentials }}"
           environment: "AzureCloud"
+          enable-AzPSSession: true
```

Comment:
*this* was the trick we were missing. Good catch!

Reply:
👍